### PR TITLE
Update travis node versions

### PR DESCRIPTION
**Pull request by mtharrison**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 language: node_js
 
 node_js:
-  - 0.10
+  - "4.0"
```
Comment (hueniverse) [inline .travis.yml]: no need for ""

Comment (nelsonic) [inline .travis.yml]: http://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Choosing-Node-versions-to-test-against

Comment (hueniverse) [inline .travis.yml]: We don't use "" anywhere else... it works just fine

Comment (nelsonic) [inline .travis.yml]: Makes sense, it *shouldn't* have to be a `string` to work with `nvm`...
I was just pointing out that travis don't *appear* to be supporting version **5** of node yet. :+1:

Comment (gergoerdosi) [inline .travis.yml]: It's not a full list, Travis supports whatever nvm supports.

**Pull request closed without merging**